Automattic/jetpack
# tools: Add changelogger release scripts

#### Changes proposed in this Pull Request:

tools/changelogger-release.sh will do `changelogger write` on a project
and all the projects it depends on.

tools/plugin-changelog-to-readme.sh will copy the latest (or a
specified) entry from CHANGELOG.md into the changelog section of
readme.txt.

#### Jetpack product discussion


p9dueE-2x5-p2

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Try the scripts out, see if they work as advertised. In particular, check that the `sed` bits work right on OSX with its ancient `sed` version.
* Anything that could be better in these to make the release process even easier?
